taker-frontend/src/components/Footer.tsx

```diff
@@ -27,6 +28,11 @@ export default function Footer() {
                     <TextDivider />
                     <Text fontSize={"20"} fontWeight={"bold"}>Contact us:</Text>
                     <SocialLinks />
+                    <TextDivider />
+                    {/* TODO Consider adding takerId */}
```

Comment:
+1 on adding the `taker_id` right away so that we know if the feedback is from a seasoned trader on ItchySats or a newbie.

Reply:
Let's just remember that this can be spoofed unless we utilise signatures.

Reply:
> +1 on adding the `taker_id` right away so that we know if the feedback is from a seasoned trader on ItchySats or a newbie.

done. I also added a small printout of the taker Id, so that anyone can refer to it when they want to debug some issues with us (if they're reaching out to us in a different way)